packages/form-builder/src/App.scss

```diff
@@ -1,3 +1,5 @@
+@import 'styles/settings.scss';
```

Comment:
fyi, I moved these styles into a new file.  This stylesheet is getting a little crazy - let's make it a point to start organizing them and take advantage of scss